Binance.Net/Enums/ConvertOrderStatus.cs

```diff
@@ -0,0 +1,31 @@
+﻿using CryptoExchange.Net.Attributes;
+
+namespace Binance.Net.Enums
+{
+    /// <summary>
+    /// Convert Order status
+    /// </summary>
+    public enum ConvertOrderStatus
+    {
+        /// <summary>
+        /// Process
+        /// </summary>
+        [Map("PROCESS")]
```

Comment:
The idea of this attribute is that you can use the EnumConverter for deserialization. So I think the `ConvertOrderStatusConverter` isn't needed